### PR TITLE
Update git ignore

### PR DESCRIPTION
**Pull request by DanielHabib**
## Summary

Set up a solid `.gitignore` so we don't have to worry about committing the wrong files

Inspired by https://github.com/github/gitignore/blob/master/Swift.gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,24 @@
-#Ignore files
-
-.DS_Store
```
Comment (demoleas21) [inline .gitignore:3]: Why did you get rid of `.DS_Store`?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+*.ipa
+timeline.xctimeline 
+playground.xcworkspace
+.build/
+Pods/ 
+Carthage/Build
+fastlane/report.xml 
+fastlane/Preview.html 
+fastlane/screenshots
+fastlane/test_output
```
Comment (demoleas21) [inline .gitignore:24]: Also you should add the files that get created when you run `pod install`
- `app-steady.xcworkspace`
- `Podfile.lock`

File: app-steady/.gitignore [hunk 3]
```diff
@@ -1 +0,0 @@
-xcuserdata
```
Comment (demoleas21) [inline app-steady/.gitignore]: Why do we need this line removed, shouldn't we just remove the file completely?

Comment (DanielHabib) [inline app-steady/.gitignore]: It is removed

File: .gitignore (hunk 1 above)
Comment (DanielHabib) [inline .gitignore:3]: I have `.DS_Store` in my global gitignore. We can leave it here also I'll put it back

**Comment by demoleas21** [comment]
👍

**Pull request merged**